rachelekm/vegishphilly
# Add owner field to Restaurants

## Overview

This PR resolves issue #7 : create a way for users to define themselves as restaurant "owners" with the intention of verified "owners" being able to add/edit their restaurant details as well as prohibit "owners" from rating their own restaurant.  Creating a ManyToMany field on the Restaurant model allows a restaurants to have multiple owners or have one user own multiple restaurants.

Connects #7 

### Notes

PR #13 now encompasses these commits in addition to more holistic "create a new user/owner" backend functionality, and will supersede this PR. This PR will close.

## Testing Instructions

 * `/scripts/update`
 * `/scripts/manage test` confirm tests pass
 * `/scripts/server` and open `localhost:8033/admin/` in browser
 * Confirm owner field in api > Restaurants
 * Add user as owner and confirm changes are saved
